src/website/views/javascript/drawer.pug

```diff
@@ -144,7 +144,31 @@ section.a-table.a-table__options.-bordered.-my--3
               | Lets the developer referer the target programmatically instead of by data-target attribute.
               | It accepts Element object and xpath selector string.
 
-h4 Preventing memory leaks
+h3.mt--2 Methods
+.p.-text.-mb--3
+  .p-text There are three 
+    b drawer methods 
+    | that you can apply to DOM elements.
+  h4 show() =&gt; void
+  p.-ml--2 Triggeres drawer 
+    b show event
+    br
+    | returns 
+    b type void
+  h4 hide() =&gt; void
+  p.-ml--2 Triggeres drawer 
+    b hide event
+    br
+    | returns 
+    b type void
+  h4 toggle() =&gt; void
+  p.-ml--2 Triggeres drawer 
+    b show/hide  events
+    br
+    | returns 
+    b type void
+
+h3 Preventing memory leaks
```

Comment:
**Preventing memory leaks** is h4 in all documentation pages but here. Please use it consistently. @jllr, do you think we should change all Preventing memory leaks to h3, or keep them as h4?

Reply:
@dani-cl-madrid Keep them as h4